src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/ManagedUpdateSiteJenkinsTest.java

```diff
@@ -99,78 +94,6 @@ private File getResource(String name) throws URISyntaxException, FileNotFoundExc
         return new File(url.toURI());
     }
     
-    public void testDoPostBack() throws URISyntaxException, IOException, GeneralSecurityException

```

Comment:
new jenkins versions removed 
```
WebRequestSettings wrs = new WebRequestSettings(
 -                    new URL(String.format("%s%s/byId/%s/postBack",
 -                            wc.getContextPath(),
 -                            Jenkins.getInstance().getUpdateCenter().getSearchUrl(),
 -                            target.getId()
 -                    )),
 -                    HttpMethod.POST
 -            );
```

so all the test don't work. I can't find an alternative to WebRequestSettings